packages/sui-ci/README.md

```diff
@@ -1,19 +1,46 @@
 # sui-ci
 
-> CLI with some useful commands for CI environments
+> CLI with some useful commands for Travis CI
 
 It provides:
-* A way to update a GitHub Commit Status from a CI environment.
+* A way to update a GitHub Commit Status from a CI environment with `update-commit-status` command.
+* A way to publish through CI all packages that used `sui-mono` with `release` command.
 
-## Requirements
+## release
+
+### Requirements
+
+The following environment variables are needed:
+* `GITHUB_TOKEN`: GitHub Personal Access Token of the user that will create the commit with the release.
+* `TRAVIS_PULL_REQUEST`: The pull request number if the current job is a pull request, `false` if it's not a pull request.
+ 
+Other optional enviromnet variables are:
+* `GITHUB_USER`: GitHub username of the user that will create the commit with the release.
+* `GITHUB_EMAIL`: GitHub email of the user that will create the commit with the release.
+
+
+### Usage
+
+The **recommended way** to use this is adding in your `travis.yml` as latest step of the `script` lifecycle the next command `npx @s-ui/ci@beta release`:
```

Comment:
`npx @s-ui/ci@beta release` shouldn't it be without beta?

Reply:
Fixed, thanks @jcger!